### PR TITLE
tools: update the usage of region startkey to region keys

### PR DESCRIPTION
**Pull request by IcePigZDB**
Signed-off-by: IcePigZDB <icepigzdb@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Support endKey in http interface `regions/key` and adjust the pdctl region command `region startKey` to `region keys`accordingly. This is originally proposed as a precondition of the push down optimisation of `infomation_schema.tikv_region_status`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: pingcap/docs-cn/pull/7637
- Other reference link(s):
    * tracking issue: tikv/pd/issues/4400

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by nolouch — APPROVED** [review]

File: pd-control.md [hunk 1]
```diff
@@ -558,20 +558,36 @@ Usage:
 }
 ```
 
-### `region startkey [--format=raw|encode|hex] <key> <limit>`
+### `region keys [--format=raw|encode|hex] <start_key> <end_key> <limit>`
 
-Use this command to query all Regions starting from a key.
+Use this command to query all Regions in a given range [startkey, endkey). It supports range without endKey, the default value of limit is 16 without endKey and -1 (no limit)  with endKey.
```
Comment (TomShawn) [inline pd-control.md:563]:
```suggestion
Use this command to query all Regions in a given range `[startkey, endkey)`. Ranges without `endKey`s are supported. With an `endKey`, the default value of `limit` is `16`. Without an `endKey`, the default value of `limit` is `-1`, which means no limit. 
```

File: pd-control.md [hunk 2]
```diff
@@ -558,20 +558,36 @@ Usage:
...
-Use this command to query all Regions starting from a key.
+Use this command to query all Regions in a given range [startkey, endkey). It supports range without endKey, the default value of limit is 16 without endKey and -1 (no limit)  with endKey. 
 
 Usage:
 
-{{< copyable "" >}}
-
 ```bash
->> region startkey --format=raw abc
+>> region keys --format=raw a         // Display all Regions start from a with default limit count 16
```
Comment (TomShawn) [inline pd-control.md:568]:
```suggestion
>> region keys --format=raw a         // Display all Regions that start from key a with the default limit count of 16
```

File: pd-control.md [hunk 3]
```diff
@@ -558,20 +558,36 @@ Usage:
...
 {
   "count": 16,
   "regions": [......],
 }
+
+>> region keys --format=raw a "" 20   // Display all Regions start from a with limit count 20
```
Comment (TomShawn) [inline pd-control.md:574]:
```suggestion
>> region keys --format=raw a "" 20   // Display all Regions that start from the key a with the limit count of 20
```

File: pd-control.md [hunk 4]
```diff
@@ -558,20 +558,36 @@ Usage:
...
+{
+  "count": 20,
+  "regions": [......],
+}
+
+>> region keys --format=raw a z      // Display all Regions in range [a, z) without limit count
```
Comment (TomShawn) [inline pd-control.md:580]:
```suggestion
>> region keys --format=raw a z      // Display all Regions in the range [a, z) without a limit count
```

File: pd-control.md [hunk 5]
```diff
@@ -558,20 +558,36 @@ Usage:
...
+{
+  "count": 16,
+  "regions": [......],
+}
+
+>> region keys --format=raw a z 20   // Display all Regions in range [a, z) with limit count 20
```
Comment (TomShawn) [inline pd-control.md:586]:
```suggestion
>> region keys --format=raw a z 20   // Display all Regions in the range [a, z) with a limit count of 20
```

**Comment by IcePigZDB** [comment]
All comments have been resolved.  PR tikv/pd/pull/4674 change the default value of `limit` with EndKey to 16 too. I have open the corresponding Chinese document modification PR pingcap/docs-cn/pull/8607 , I will open the English version after thie PR merged and notify you, thanks very much.

**Comment by TomShawn** [comment]
> @TomShawn All comments have been resolved. PR [tikv/pd/pull/4674](https://github.com/tikv/pd/pull/4674) change the default value of `limit` with EndKey to 16 too. I have open the corresponding Chinese document modification PR [pingcap/docs-cn/pull/8607](https://github.com/pingcap/docs-cn/pull/8607) , I will open the English version after thie PR merged and notify you, thanks very much.

Thanks! If the default value of `limit` with EndKey is changed to 16, maybe you can change the value right in this document.

**Comment by IcePigZDB** [comment]
> > @TomShawn All comments have been resolved. PR [tikv/pd/pull/4674](https://github.com/tikv/pd/pull/4674) change the default value of `limit` with EndKey to 16 too. I have open the corresponding Chinese document modification PR [pingcap/docs-cn/pull/8607](https://github.com/pingcap/docs-cn/pull/8607) , I will open the English version after thie PR merged and notify you, thanks very much.
> 
> Thanks! If the default value of `limit` with EndKey is changed to 16, maybe you can change the value right in this PR after the docs-cn PR 8670 is merged.

Done.

**Comment by TomShawn** [comment]
@IcePigZDB https://github.com/pingcap/docs-cn/pull/8607 has been merged. Please sync it to this PR. Thanks!

**Comment by IcePigZDB** [comment]
> @IcePigZDB [pingcap/docs-cn#8607](https://github.com/pingcap/docs-cn/pull/8607) has been merged. Please sync it to this PR. Thanks!
I checked again. It's synchronized in previous commit.

File: pd-control.md [hunk 6]
```diff
@@ -558,20 +558,36 @@ Usage:
 }
 ```
 
-### `region startkey [--format=raw|encode|hex] <key> <limit>`
+### `region keys [--format=raw|encode|hex] <start_key> <end_key> <limit>`
 
-Use this command to query all Regions starting from a key.
+Use this command to query all Regions in a given range `[startkey, endkey)`. Ranges without `endKey`s are supported. The default value of `limit` is `16`, set to `-1` for unlimited.
```
Comment (TomShawn) [inline pd-control.md:563]: The value of `-1` means no limit.

Comment (TomShawn) [reply pd-control.md:563]:
```suggestion
Use this command to query all Regions in a given range `[startkey, endkey)`. Ranges without `endKey`s are supported. The default value of `limit` is `16`, and the value of `-1` means unlimited. 
```

Comment (TomShawn) [reply pd-control.md:563]: @nolouch PTAL, thanks~

**Pull request merged**